network/network_test.go

```diff
@@ -251,6 +251,33 @@ func TestNetwork(t *testing.T) {
 		assert.Equal(t, networkX.ReachabilityStatus(), "Private")
 	})
 
+	t.Run("Supported Protocols", func(t *testing.T) {
+		protosNetB := networkB.Protocols()
```

Comment:
Kay, keep only one, for example Bootstrap one.

Reply:
Fixed.